src/lib.rs

```diff
@@ -208,6 +222,18 @@ impl<'a> Ammonia<'a> {
             child.parent.replace(None);
             dom.append(&parent.clone(), NodeOrText::AppendNode(child.clone()));
         }
+        pass
+    }
+
+    fn fix_child(&self, child: &mut Handle, link_rel: &Option<StrTendril>) {
```

Comment:
Because if I do the conversion inside `fix_child`, then it needs to happen for every element, and I prefer to do it once. The conversion involves copying.